pyvo/registry/regtap.py

```diff
@@ -837,7 +837,8 @@ def describe(self, verbose=False, width=78, file=None):
         if verbose:
             if self.source_value:
                 print(f"\nSource: {self.source_value}", file=file)
-            if self.creators:
+            # don't print creators if its first value is overly long
+            if self.creators and len(self.creators[0]) < width:
```

Comment:
Hm... I suspect this will one of these days confuse someone.  Can't we do something like self.creators[0][:70] or something?  Perhaps with a "..." appended if self.creators[0] is longer than 70 chars?  I give you it'll (mostly) be junk anyway, but most of the time it's better to show junk than to *pretend* it's not even there.

I mention in passing that  Uthamadhanapuram Venkatasubbaiyer Swaminatha Iyer is the actual name of a famous Tamil linguist...

Reply:
omg didn't think of such long names. I'll think about something

Reply:
should work with the new commit, I applied your suggestion of truncating at 70 characters and appending `...`